sat/main.py

```diff
@@ -119,7 +120,11 @@ def main():
                          args.command, args.command)
             sys.exit(1)
 
-        subcommand(args)
+        try:
+            subcommand(args)
+        except InvalidGrantError as err:
+            LOGGER.error("Invalid grant error: The token is not active or is invalid."
```

Comment:
IMO, it's unnecessary to say "Invalid grant error" to the user as it's probably not clear what that means. The rest of the message is sufficient, IMO. It also needs a space at the end of this string to ensure there is a space after "invalid." and before "Please"

```suggestion
            LOGGER.error("The token is not active or is invalid. "
```